tools/isa-info.c

```diff
@@ -70,6 +70,10 @@ int main(int argc, char** argv) {
 	printf("\tAVX512VP2INTERSECT: %s\n", cpuinfo_has_x86_avx512vp2intersect() ? "yes" : "no");
 	printf("\tAVX512_4VNNIW: %s\n", cpuinfo_has_x86_avx512_4vnniw() ? "yes" : "no");
 	printf("\tAVX512_4FMAPS: %s\n", cpuinfo_has_x86_avx512_4fmaps() ? "yes" : "no");
+	
+	//avx10.1 (in future will add other avx10 version)
+	printf("\tAVX10_1: %s\n", cpuinfo_has_x86_avx10_1() ? "yes" : "no");
+
```

Comment:
remove blank lines.  avx10_1 is an ISA and should be grouped with the rest

Reply:
CI have linter automation that shoudl detect this